tests/unit/job/test_copy.py

```diff
@@ -70,6 +76,20 @@ def test_to_api_repr_with_encryption_none(self):
             resource, {"copy": {"destinationEncryptionConfiguration": None}}
         )
 
+    def test_operation_type_unspecified(self):
+        from google.cloud.bigquery.job import OperationType
+
+        config = self._make_one()
+        self.assertEqual(
+            config.operation_type, OperationType.OPERATION_TYPE_UNSPECIFIED
+        )
```

Comment:
This part seems like it belongs in a (not-yet-there) `test_ctor_defaults` testcase, along with assertions about other default properties.